draft-ietf-tls-tls13.md

```diff
@@ -2757,10 +2758,16 @@ Structure of this message:
        opaque DistinguishedName<1..2^16-1>;
 
        struct {
+           opaque certificate_extension_oid<0..2^8-1>;
+           opaque certificate_extension_values<0..2^8-1>;
+       } CertificateExtension
+
+       struct {
            ClientCertificateType certificate_types<1..2^8-1>;
            SignatureAndHashAlgorithm
              supported_signature_algorithms<2..2^16-2>;
            DistinguishedName certificate_authorities<0..2^16-1>;
```

Comment:
What would you think about pushing this up into the ServerHello.?That would support client-initiated client auth.